Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.xaml

```diff
@@ -0,0 +1,313 @@
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="CommonStates">
+                                <VisualState x:Name="Normal" />
+                                <VisualState x:Name="ReadOnly" />
+                                <VisualState x:Name="MouseOver" />
+                                <VisualState x:Name="Disabled" />
+                                <VisualState x:Name="Enabled" />
+                            </VisualStateGroup>
+                            <VisualStateGroup x:Name="HeaderStates">
+                                <VisualState x:Name="HeaderExpanderMouseOver" />
+                                <VisualState x:Name="HeaderSelectorMouseOver" />
+                                <VisualState x:Name="HeaderExpanderPressed" />
+                                <VisualState x:Name="HeaderSelectorPressed" />
+                            </VisualStateGroup>
+                            <VisualStateGroup x:Name="ContentStates">
+                                <VisualState x:Name="ContentExpanded">
+                                    <VisualState.Setters>
+                                        <Setter Target="PART_MainContentRow.Height" Value="Auto" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="ContentCollapsed">
+                                    <VisualState.Setters>
+                                        <Setter Target="PART_MainContentRow.Height" Value="0" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                        </VisualStateManager.VisualStateGroups>
+                        <Grid Grid.RowSpan="2" VerticalAlignment="Top">
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto" />
+                                <RowDefinition Height="Auto" x:Name="PART_MainContentRow" />
+                            </Grid.RowDefinitions>
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="Auto" />
+                                <ColumnDefinition />
+                            </Grid.ColumnDefinitions>
+                            <ToggleButton x:Name="PART_ExpanderToggleButton" Height="40"
+                                          Style="{StaticResource HeaderToggleButtonStyle}" VerticalAlignment="Top"
+                                          IsChecked="{Binding IsExpanded, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}"
+                                          Width="40">
+                                <SymbolIcon Symbol="Forward" />
+                            </ToggleButton>
+                            <Grid x:Name="PART_Header" VerticalAlignment="Stretch"
```

Comment:
Sorry I wasn't more clear. The togglebutton of the expander style would have the ContentTemplate, and Content set to the properties of the control
```
<ToggleButton x:Name="PART_ExpanderToggleButton" 
                Content="{TemplateBinding Header}"
                ContentTemplate="{TemplateBinding HeaderTemplate}"
                IsChecked="{Binding IsExpanded, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}"
                Style="{StaticResource HeaderToggleButtonStyle}"/>
```

The template of the ToggleButton wold then look like:
```
<Grid>
    <!-- Visual states -->
    <Grid.ColumnDefinitions>
        <ColumnDefinition Width="Auto"/>
        <ColumnDefinition Width="*"/>
    </Grid.ColumnDefinitions>
    <SymbolIcon x:Name="Arrow" Width="40"
                RenderTransformOrigin="0.5,0.5"
                Symbol="Forward">
        <SymbolIcon.RenderTransform>
            <RotateTransform />
        </SymbolIcon.RenderTransform>
    </SymbolIcon>
    <ContentPresenter x:Name="ContentPresenter" AutomationProperties.AccessibilityView="Raw"
                        Grid.Column="2"
                        BorderBrush="{TemplateBinding BorderBrush}"
                        BorderThickness="{TemplateBinding BorderThickness}"
                        ContentTemplate="{TemplateBinding ContentTemplate}"
                        ContentTransitions="{TemplateBinding ContentTransitions}"
                        Content="{TemplateBinding Content}"
                        HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
                        Padding="{TemplateBinding Padding}"
                        VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}">
    </ContentPresenter>
</Grid>
```

Note that if alignment of the content is too close to the arrow then add a small margin to the content presenter